#! /bin/sh /usr/share/dpatch/dpatch-run
## randn_doc.dpatch by Thomas Weber <thomas.weber.mail@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Better explanation of randn()'s return value (fix for #477349)

@DPATCH@
diff -urNad octave3.0-3.0.1~/src/DLD-FUNCTIONS/rand.cc octave3.0-3.0.1/src/DLD-FUNCTIONS/rand.cc
--- octave3.0-3.0.1~/src/DLD-FUNCTIONS/rand.cc	2008-04-21 18:00:20.000000000 +0200
+++ octave3.0-3.0.1/src/DLD-FUNCTIONS/rand.cc	2008-05-21 12:09:44.000000000 +0200
@@ -471,8 +471,9 @@
 @deftypefnx {Loadable Function} {} randn (@var{n}, @var{m})\n\
 @deftypefnx {Loadable Function} {} randn (\"state\", @var{x})\n\
 @deftypefnx {Loadable Function} {} randn (\"seed\", @var{x})\n\
-Return a matrix with normally distributed random elements. The\n\
-arguments are handled the same as the arguments for @code{rand}.\n\
+Return a matrix with normally distributed pseudo-random\n\
+elements having zero mean and variance one. The arguments are\n\
+handled the same as the arguments for @code{rand}.\n\
 \n\
 By default, @code{randn} uses a Marsaglia and Tsang Ziggurat technique to\n\
 transform from a uniform to a normal distribution. (G. Marsaglia and\n\
